.github/ISSUE_TEMPLATE/propose-process-change.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Do you want to propose a change to the EIP process itself?
```

Comment:
Why is this one not in the `ISSUE_TEMPLATE/config.yml`?  Does this still show up in the list of options along with the options in `config.yml`?

Reply:
Yes, it does. This isn't in the `ISSUE_TEMPLATE/config.yml` because it's an issue template, not an external link.